docs/_static/css/light.css

```diff
@@ -0,0 +1,31 @@
+/* links */
+.rst-content a{
+    color: #002fa7;
+}
+
+.rst-content a:visited {
+    color: #002fa7;
+}
+
+.rst-content .highlighted {
+	background: #eac545;
+}
+
+/* code blocks highlights */
+.rst-content pre {
+	background: #e5e5e5;
+}
+
+.wy-side-nav-search img{
+	width: 80px !important;
+}
```

Comment:
> Hmm...I couldn't figure out the "empty space" scenario.

- The `<a>` element containing the logo has a bottom margin of `0.809em` that could be removed.
- Then the `<div>` that contains that link has `0.809em` of padding.
- Something adds padding of `5px` around `.wy-side-nav-search img` - if you remove it, logo gets slightly bigger.
- `.wy-side-nav-search > div.version` has a negative top margin. If the above are not enough, you could use that to get version as close to the logo as needed.

I think that the above bring us pretty close to the spacing in the design.